documentation/documentation.gradle.kts

```
@@ -56,6 +57,7 @@ dependencies {
 asciidoctorj {
 	modules {
 		diagram.use()
+		pdf.version("1.5.3")
```

Please define this version as `asciidoctor-pdf.version=1.5.3` in `gradle.properties` and reference it as follows here.

```suggestion
		pdf.version(versions["asciidoctor-pdf"])
```

---

No problem 😄 Hope it is better now ?